aws_jobs/cyhy-data-retriever.py

```diff
@@ -45,18 +47,22 @@ def main():
     # Read parameters in from config file
     config = SafeConfigParser()
     config.read([args['--config']])
-    CLIENT_PRIVATE_KEY_FILE = config.get('DEFAULT', 'CLIENT_PRIVATE_KEY_FILE')
```

Comment:
I removed this because it isn't used anywhere.  Can it be removed from the example config as well?

Reply:
I think so, but get the go-ahead from @KyleEvers